src/github-cli/install.sh

```diff
@@ -10,7 +10,7 @@
 CLI_VERSION=${VERSION:-"latest"}
 INSTALL_DIRECTLY_FROM_GITHUB_RELEASE=${INSTALLDIRECTLYFROMGITHUBRELEASE:-"true"}
 
-GITHUB_CLI_ARCHIVE_GPG_KEY=C99B11DEB97541F0
+GITHUB_CLI_ARCHIVE_GPG_KEY=23F3D4EA75716059
```

Comment:
Can we bump the patch version here to 1.0.4? We follow semantic versioning. https://github.com/devcontainers/features/blob/main/src/github-cli/devcontainer-feature.json#L3

Features are released every time a PR is merged, these changes won't take effect unless the version specified here is different than the ones already published 😄

Reply:
Thanks for letting me know!